cohortextractor/study_definition.py

```diff
@@ -313,7 +352,11 @@ def make_df_from_expectations(self, population):
         # its incidence calculated as a mask
         for colname, dtype in self.pandas_csv_args["dtype"].items():
             definition_args = self.pandas_csv_args["args"][colname]
-            if definition_args["funcname"] == "aggregate_of":
+            if definition_args["funcname"] in [
+                "aggregate_of",
+                "with_value_from_file",
+                "which_exist_in_file",
```

Comment:
very minor nitpick; I'd make this list a variable so we don't have to repeat the actual function names.

Reply:
I wasn't sure about this, either 🙂 Specifically, I wasn't sure how to distinguish `aggregate_of` from `with_value_from_file` and `which_exist_in_file`; we always use the last two, together; and sometimes use all three, together. So, how to name these variables? I couldn't quickly think of good names, so repeated the actual function names.

We could call them according to the types of function they represent, like this:

```python
aggregation_functions = ["aggregate_of"]
user_file_functions = ["with_value_from_file", "which_exist_in_file"]
```

But these names don't really communicate why we're handling these types of function differently.